src/Unused/Cache/FindArgsFromIgnoredPaths.hs

```diff
@@ -0,0 +1,48 @@
+module Unused.Cache.FindArgsFromIgnoredPaths
+    ( findArgs
+    ) where
+
+import Data.Char (isAlphaNum)
+import Data.List (isSuffixOf)
+import System.FilePath
+
+findArgs :: [String] -> [String]
+findArgs =
+    concat <$> (fmap . fmap) ignoreToFindArgs validIgnoreOptions
```

Comment:
As discussed in slack, pretty sure this can be: `concatMap ignoreToFindArgs . validIgnoreOptions` instead, which is much more readable IMO.

```haskell
λ> let foo x = [x]
λ> :t foo
foo :: t -> [t]
λ> let bar a@(x:xs) = a
λ> :t bar
bar :: [t] -> [t]
λ> :t concatMap foo . bar
concatMap foo . bar :: [b] -> [b]
```